server/go.mod

```diff
@@ -72,11 +72,11 @@ require (
 	github.com/wiggin77/merror v1.0.5
 	github.com/xtgo/uuid v0.0.0-20140804021211-a0b114877d4c
 	github.com/yuin/goldmark v1.7.8
-	golang.org/x/crypto v0.32.0
+	golang.org/x/crypto v0.35.0
```

Comment:
Unrelated, but what is the reason for these go.mod changes?

Reply:
It was the SNYK action was complaining about this. I don't know why, but the vulnerability seemed legit to me.

Reply:
@agnivade here is the check for context: https://app.snyk.io/org/mattermost-core/project/6e011df4-086c-4494-80fe-d44791c8551a/pr-check/ab5e05ec-1b2c-4bcc-8371-083ec361ef9c